### ISSUE TITLE
SENet Training on ImageNet

### ISSUE THREAD

**Issue by ahkarami** [moskomule/senet.pytorch#5]
Dear @moskomule,
Just as a guide. If you need appropriate machine (i.e., GPU) for training the SENet on ImageNet, maybe Soumith Chintala can help you. I mean that the PyTorch team welcome to porting important CNN models into the Torchvision Models. You can see [NASNet Model](https://github.com/pytorch/vision/issues/321#issuecomment-342207053).

**Comment by moskomule** [comment]
Thank you. Now I'm training on my environment but it uses almost all of my computational resource. If possible I'd like to ask him (I don't know if he likes to train SENet or not)...

**Comment by karandwivedi42** [comment]
@moskomule Did you get the training results?

Also, Thanks for the repo :)

**Comment by moskomule** [comment]
Hi, still it is training now. Since the authors use many GPU while I have two so the learning rate and other parameters are different so it looks not good so far.

**Comment by moskomule** [comment]
The first run (more than a week) finished and cannot achieve the original rate. It may be because of the difference between the authors' and my batch-size and learning rate.
